### PR TITLE
[Robot] Verify preferred email and phone settings

### PR DESCRIPTION
**Pull request by rhakeshj**
Verify preferred email and phone settings
Fixed EDA settings page link in existing tests to redirect to Education Cloud Settings

WI - W-9294254, W-9294235

# Critical Changes

# Changes

# Issues Closed

# New Metadata

## Unpackaged Metadata

# Deleted Metadata

# Testing Notes

### CODE REVIEW

**Review by skosgi1 — APPROVED** [review]
Changes to the lwc html files look good.

File: robot/EDA/tests/browser/settings/account_model/admin_household.robot
Comment (Safiyab) [inline robot/EDA/tests/browser/settings/account_model/admin_household.robot:67]: Will the "Tell Me More" string change for different settings? If it is same, you can hardcode it in the keyword. But I'll leave it to you since you know if this info is important to be kept in the robot script as well.

File: robot/EDA/tests/browser/settings/contact_information/preferred_email_phone.robot
Comment (Safiyab) [inline robot/EDA/tests/browser/settings/contact_information/preferred_email_phone.robot:13]: This is part of the `Test Setup`, do you need to add it again in the test case?

**Review by Safiyab — APPROVED** [review]
You make everything look so simple. Great work! :)

File: robot/EDA/tests/browser/settings/account_model/admin_household.robot
Comment (rhakeshj) [reply robot/EDA/tests/browser/settings/account_model/admin_household.robot:67]: Right now it doesn't change. I just kept it open for any future help link validations as new EDA settings page is evolving now

File: robot/EDA/tests/browser/settings/contact_information/preferred_email_phone.robot
Comment (rhakeshj) [reply robot/EDA/tests/browser/settings/contact_information/preferred_email_phone.robot:13]: Yes. I still have few more tests to be added to this contact information settings.

**Review by spinkelman — APPROVED** [review]
After Functional Review, This is approved.

**Pull request merged**
